src/test/java/io/pravega/connectors/flink/EventTimeOrderingFunctionTest.java

```diff
@@ -104,15 +100,15 @@ public void testOrdering() throws Exception {
     @Test
     public void testLateElements() throws Exception {
```

Comment:
This test does not really test late events in such case because there are no data triggering the timestamp moving forward before the watermark. We need to change this test a little bit.
First we can let an on-time data go in the function, and then comes the watermark, check the output, and then process another late data to see if it is discarded.

Reply:
Updated.